scipy/stats/stats.py

```diff
@@ -2858,13 +2893,124 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         med = np.apply_over_axes(center, arr, axis)
         mad = np.median(np.abs(arr - med), axis=axis)
 
-    return scale * mad
+    # NOTE: _scale_conversions was for IQR so for MAD we need to divide by 2
+    if is_str and scale_key == 'normal':
+        scale /= 2
+
+    return mad / scale
+
+
+_median_absolute_deviation_deprec_msg = """\
+The function `median_absolute_deviation` is deprecated.
+To preserve the existing behavior of the program, use
+`scipy.stats.median_abs_deviation(..., scale=1.4826)`.
+The value 1.4826 is not numerically precise for scaling
+with a normal distribution. For a numerically precise value, use
+`scipy.stats.median_abs_deviation(..., scale='normal')`.
+"""
+
+
+@np.deprecate(old_name='median_absolute_deviation',
+              new_name='median_abs_deviation',
+              message=_median_absolute_deviation_deprec_msg)
+def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
+                              nan_policy='propagate'):
+    r"""
+    Compute the median absolute deviation of the data along the given axis.
+
+    The median absolute deviation (MAD, [1]_) computes the median over the
+    absolute deviations from the median. It is a measure of dispersion
+    similar to the standard deviation but more robust to outliers [2]_.
+
+    The MAD of an empty array is ``np.nan``.
+
+    .. versionadded:: 1.3.0
+
+    Parameters
+    ----------
+    x : array_like
+        Input array or object that can be converted to an array.
+    axis : int or None, optional
```

Comment:
If you look at the output, it's broken:

https://19378-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.stats.median_absolute_deviation.html#scipy.stats.median_absolute_deviation

My guess is that it's the newlines in `_median_absolute_deviation_deprec_msg`. I would try getting rid of those and doing `make html-scipyorg` in `doc/` locally and check to see if it fixes things. If it doesn't, next I would remove the `np.deprecate` altogether to rule that out.

Sometimes there are errant empty unicode spaces that can mess things up if you did a copy-paste from rendered GitHub HTML (?), but I pulled your branch locally and don't see them so I doubt that's it.

I can look tomorrow on my machine where I can actually properly build SciPy if you can't figure it out in the meantime.

Reply:
`np.deprecate` is the problem:

https://github.com/numpy/numpy/pull/16349

I think we can work around this issue by unindenting the whole docstring. I'll try and push if it works.